spaceward/src/utils/web3-onboard/wagmi/index.ts

```
@@ -0,0 +1,309 @@
+		return wagmiConnectors.find((connector) => connector.name === label);
+	} catch (e) {
+		console.error("error disconnecting wallet from wagmi", e);
+	}
+}
+
+const convertW3OToWagmiWallet = (
+	label: string,
+	provider: EIP1193Provider,
+	coreProviderMethods: WagmiInitOptions,
+): Connector => {
+	if (!coreProviderMethods || typeof coreProviderMethods === undefined) {
```

Biome

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal
Unsafe fix: Compare the result of `typeof` with a valid type name


(lint/suspicious/useValidTypeof)